packages/react-focus/src/components/FocusZone/FocusZonePage.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
```

Comment:
We don't have to change this now but we should consider whether we actually want these to live inside the relevant package (which will definitely create a circular dep just for example stuff if OUFR eventually depends on this new package) or inside fabric-website or something. 

If we made that change everywhere, it would also let us reduce the size of published packages. It would be a big change but definitely something that could be worth doing (and which I'd be interested in doing).

Reply:
yes! I was thinking... should these be in umm, react-focus-docs

Reply:
A dedicated docs package for each published package seems like a bit much. Though if you put them all in the same package, once we multiple components by the same name, you have to figure out how to distinguish them, so...not sure of the best approach here.

Reply:
Thinking about this again, the simplest solution would be to have the examples in a docs or website package, with sub-folders per package (which solves the issue of how to have examples for both e.g. OUFR button and experiments button). Definitely not happening in this PR though.